.github/workflows/e2e_tests.yml

```diff
@@ -381,10 +381,9 @@ jobs:
 
       - name: run release builder
         env:
-          BINTRAY_USER: ${{ secrets.BINTRAY_USER }}
-          API_KEY: ${{ secrets.BINTRAY_API_KEY }}
+          CLOUDSMITH_USER: ${{ secrets.CLOUDSMITH_USER }}
           TAG: ${{ steps.get_tag.outputs.TAG }}
-          BINTRAY_API: ${{ secrets.BINTRAY_ARCHIVE_API }}
+          CLOUDSMITH_API_KEY: ${{ secrets.CLOUDSMITH_API_KEY }}
```

Comment:
Shouldn't this be the Archive URL?

Reply:
We dont need an URL since we are not using the REST api with curl, instead we are using the cli client to push the packages